NEWS.txt

```diff
@@ -3,243 +3,29 @@ Release Notes
 =============
 
 
-Theano 1.0.0 (15th of November, 2017)
-=====================================
+Theano 1.0.1 (1st of December, 2017)
+====================================
 
-This is a final release of Theano, version ``1.0.0``, with a lot of
-new features, interface changes, improvements and bug fixes.
+This is a release of Theano, version ``1.0.1``, with important bug fixes.
 
 We recommend that everybody update to this version.
 
-Highlights (since 0.9.0):
- - Announcing that `MILA will stop developing Theano <https://groups.google.com/d/msg/theano-users/7Poq8BZutbY/rNCIfvAEAwAJ>`_
- - conda packages now available and updated in our own conda channel ``mila-udem``
-   To install it: ``conda install -c mila-udem theano pygpu``
- - Support NumPy ``1.13``
- - Support pygpu ``0.7``
- - Moved Python ``3.*`` minimum supported version from ``3.3`` to ``3.4``
- - Added conda recipe
- - Replaced deprecated package ``nose-parameterized`` with up-to-date package ``parameterized`` for Theano requirements
- - Theano now internally uses ``sha256`` instead of ``md5`` to work on systems that forbid ``md5`` for security reason
- - Removed old GPU backend ``theano.sandbox.cuda``. New backend ``theano.gpuarray`` is now the official GPU backend
- - Make sure MKL uses GNU OpenMP
+Highlights (since 1.0.0):
 
-   - **NB**: Matrix dot product (``gemm``) with ``mkl`` from conda
-     could return wrong results in some cases. We have reported the problem upstream
-     and we have a work around that raises an error with information about how to fix it.
+ - Fixed compilation and improved float16 support for topK on GPU
 
- - Improved elemwise operations
+   - **NB**: topK support on GPU is experimental and may not work for large input sizes on certain GPUs
 
-   - Speed-up elemwise ops based on SciPy
-   - Fixed memory leaks related to elemwise ops on GPU
+ - Fixed cuDNN reductions when axes to reduce have size ``1``
+ - Attempted to prevent re-initialization of the GPU in a child process
+ - Fixed support for temporar paths with spaces in Theano initialization
+ - Whole spell checking
```

Comment:
What does this mean?

Reply:
Here: https://github.com/Theano/Theano/pull/6525

Reply:
Then it should be phrased like this: `Spell check pass on the documentation`

Otherwise it's not a sentence and doesn't make sense.